mkdocs.yml

```diff
@@ -56,6 +56,8 @@ plugins:
   - exclude:
       glob:
         - README.md
+        - "*.gotmpl"
```

Comment:
@zalsader do you want to make this contribution to upstream rook? Would be easier to maintain that way, since we pull changes from rook upstream regularly.

Reply:
It is on upstream rook and on the master branch. When the file was changed, these were forgotten.

Reply:
Aah i see, sorry my bad